README.md

```diff
@@ -0,0 +1,41 @@
+# @artsy/library-example
+
+Example repo for publishing `@artsy` modules to NPM. Duplicate this repo, update the `package.json` package name, and be good to go!
+
+## Commands
+
+```bash
+yarn test
+yarn type-check
+yarn lint
+yarn compile
+yarn watch
+```
+
+## Publishing Overview
+
+There are a few key tools that we use for our publishing pipeline. Notably
+
+- [Auto](https://github.com/intuit/auto): Generate releases based on semantic version labels on pull requests
+- [Artsy's CircleCI Orbs](https://github.com/artsy/orbs): Orbs are packages of CircleCI configuration that can be shared across projects
+
+Combining these things together, we get seamless DX around releases that can be triggered directly from PRs via labels, and minimal CI config for running common script commands in an ideal way. Additionally, an automated changelog is generated for every release.
```

Comment:
We should have a step by step for actually setting up a project.

For instance: you have to set up the project in Circle

Reply:
Ah, right. Anything else you can remember from recent memory that you had to do in icons? Or is Circle addition it?

Reply:
Oh, give Artsy repo permissions

Reply:
Updated